lemur/static/app/angular/certificates/view/view.tpl.html

```diff
@@ -173,7 +173,7 @@ <h2 class="featurette-heading">Certificates
                   </ul>
                 </uib-tab>
                 <uib-tab>
-                  <uib-tab-heading>Domains</uib-tab-heading>
+                  <uib-tab-heading>SAN</uib-tab-heading>
```

Comment:
I would do Subject Alternate Names if looks ok.

Reply:
**Subject Alternate Names** is definitely more readable than **SAN**.
The only problem is that the UI might not provide enough space for the not abbreviated version, though it might still work.

Right now, when expanding a certificate in Lemur by clicking on `MORE`, you see

Basic Info | Endpoints | Notifications | Roles | Destinations | **Domains** | Replaces

![view](https://user-images.githubusercontent.com/4578421/141179672-94c89efe-e901-4223-b670-f415e4a823b3.png)

Reply:
+1, rendering it in UI is the only issue given longer string `Subject Alternate Names`, was not the first choice for me as well otherwise.